docs/user_guides/fs/feature_group/on_demand_transformations.md

```diff
@@ -0,0 +1,180 @@
+#### Recompute all on-demand features
+
+The `compute_on_demand_features` function recomputes all on-demand features attached to a feature view and adds them to the feature vectors provided as input to the function. This function does not apply model-dependent transformations to any of the features. The `transform` function can be used to apply model-dependent transformations to the returned values if required.
+
+The `request_parameter` in this case, can be a list of dictionaries that specifies the input parameters for the computation of on-demand features for each feature vector given as input to the function or can be a dictionary if the on-demand transformations require the same parameters for all input feature vectors.
+
+=== "Python"    
+!!! example "Recomputing all on-demand features and manually applying model dependent transformations."
+    ```python
+
+    # Specify request parameters for each serving key.
+    untransformed_feature_vector = feature_view.get_feature_vector(entry={"id":1}, transformed=False)
+
+    # re-compute and add on-demand features to the feature vector
+    feature_vector_with_on_demand_features = fv.compute_on_demand_features(untransformed_feature_vector, request_parameter={"arg1":1, "arg2":2})
+
+    # Applying model dependent transformations
+    encoded_feature_vector = fv.transform(feature_vector_with_on_demand_features)
+
+    # Specify request parameters for each serving key.
+    untransformed_feature_vectors = feature_view.get_feature_vectors(entry=[{"id":1}, {"id":2}], transformed=False)
+
+    # re-compute and add on-demand features to the feature vectors - Specify unique request parameter for each feature vector
+    feature_vectors_with_on_demand_features = fv.compute_on_demand_features(untransformed_feature_vectors, request_parameter=[{"arg1":1, "arg2":2}, {"arg1":11, "arg2":12}])
+
+    # re-compute and add on-demand feature to the feature vectors - Specify common request parameter for all feature vectors
+    feature_vectors_with_on_demand_features = fv.compute_on_demand_features(untransformed_feature_vectors, request_parameter={"arg1":1, "arg2":2})
+
+    # Applying model dependent transformations
+    encoded_feature_vector = fv.transform(feature_vectors_with_on_demand_features)
+
+    ```
+
+#### Recompute one on-demand feature
+
+On-demand transformation functions can also be accessed and executed as normal functions by using the dictionary `on_demand_transformations` that maps the on-demand features to their corresponding on-demand transformation function.
+
+=== "Python"    
+!!! example "Executing each on-demand transformation function"
+    ```python
+
+    # Specify request parameters for each serving key.
+    feature_vector = feature_view.get_feature_vector(entry={"id":1}, transformed=False, return_type="pandas")
+
+    # Applying model dependent transformations
+    feature_vector["on_demand_feature1"] = fv.on_demand_transformations["on_demand_feature1"](feature_vector["arg1"], arg2)
```

Comment:
fv.on_demand_transformations is dictionary that maps on demand feature names to the on demand transformation function `fv.on_demand_transformations["on_demand_feature1"]` directly give the function used to compute `on_demand_feature1`. So it can be called directly.